pkg/combustion/templates/10-rpm-install.sh.tpl

```diff
@@ -2,6 +2,14 @@
 set -euo pipefail
 
 #  Template Fields
-#  RPMs - A string that contains all of the RPMs present in the user created config directory, separated by spaces.
+#  RepoName - name of the air-gapped repository
+#  RepoPath - path to the airgapped repository relative to the combustion dir in the image
+#  PKGList  - list of packages that will be installed
 
-rpm -ivh --nosignature {{.RPMs}}
\ No newline at end of file
+{{ if ne .RepoName "" -}}
```

Comment:
You can freely add a standalone RPM along with RPMs that need to be resolved in the RPM directory. If you do so, the standalone RPM will be wrapped in the resolved RPM repository, thus using the RPM repo installation logic. That way you will have both standalone RPMs + resolved RPMs. The non-repo solution is only when you want to add standalone RPMs without adding RPMs that need resolution, then since going through the whole resolution logic would be pointless, we just use the old logic. I actually demoed this exact case on our last review.

My line of thought here was that an RPM repo is created only if at least one of the RPMs requires resolution (has specified a third-party repo or has given a SCC reg code), otherwise I wanted to use the old logic, mainly because IMO going through the whole resolution process + creating a rpm repo for standalone rpms did not make sense to me (that said I have never done this before so I might be totally wrong here).

Nevertheless if you think that this offers too much complexity (or just plain does not make sense) I can decouple the ["repo create"](https://github.com/ipetrov117/edge-image-builder/blob/rpm-resolution/pkg/repo/repo.go#L23) logic from the ["resolution"](https://github.com/ipetrov117/edge-image-builder/blob/rpm-resolution/pkg/repo/repo.go#L24) logic and when only standalone RPMs are provided, create an RPM repo without doing resolution (would probably be done in the combustion package). This should not be a big problem. WDYT?

Reply:
Second way of installation has been removed, due to always doing RPM resolution.
Only the
```bash
zypper --no-gpg-checks install -r {{.RepoName}} -y --force-resolution --auto-agree-with-licenses {{.PKGList}}
```
installation is supported now.